.github/workflows/build-and-publish.yml

```
@@ -45,7 +45,7 @@ jobs:
           mvn versions:set -DnewVersion="$PACKAGE_VERSION"
 
       - name: Release Maven package
-        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml"
+        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml" -Dgpg.passphrase=${{ secrets.GPG_PRIVATE_KEY }}
```

Ah this will need to be the passphrase secret:

```suggestion
        run: mvn deploy --batch-mode --settings "/home/runner/.m2/settings.xml" -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
```

We can also remove that from the `env` list if we're not using it elsewhere in the step

---

I'll remove it if we can run it successfully, No guarantee that this will work.